datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -196,6 +202,7 @@ public static void configureRuntimeWiring(final RuntimeWiring.Builder builder) {
         return GraphQLEngine.builder()
                 .addSchema(schema())
                 .addDataLoaders(loaderSuppliers(LOADABLE_TYPES))
+                .addDataLoader("Aspect", (context) -> createAspectLoader(context))
```

Comment:
Interesting. We'll have to keep an eye on this. Adding an aspect specific fetcher kind of breaks the existing abstractions huh?